src/contents/en-gb/doc_xamarin.md

```diff
@@ -0,0 +1,210 @@
+# SAFE Mobile App Tutorial
+
+In this tutorial, we will create a [Xamarin.Forms](https://github.com/xamarin/Xamarin.Forms/) application. Xamarin.Forms is an open source and cross-platform tool which provides a way to quickly build native apps for iOS, Android, Windows and macOS using C# and F#.
+
+To create this app, we will use the [MaidSafe.SafeApp](https://www.nuget.org/packages/MaidSafe.SafeApp/) NuGet package which exposes the `safe_app` API to connect and interact with the SAFE Network.

```

Comment:
Does it need to say the nuget package reference is already in the solution?

Reply:
No we don't have to mentioned that. If a developer create an app from sketch then he/she have to install otherwise it is understood that the package is available in any existing app. 
Common practice for dotnet developers